data/tunein.com-view.js

```diff
@@ -0,0 +1,8 @@
+
+/**
+ * MediaKeys namespace.
+ */
+if (typeof MediaKeys == "undefined") var MediaKeys = {};
+
+MediaKeys.playButton = "//div[@class='playbutton-cont col' and not(contains(@style,'none'))]";
+MediaKeys.pauseButton = "//div[@class='playbutton-cont col' and not(contains(@style,'none'))]";
```

Comment:
Similar to my other comment, these could be: //div[contains(@class,'playbutton') and not(contains(@style,'none'))].